Validation/RecoTrack/python/plotting/trackingPlots.py

```diff
@@ -487,12 +493,16 @@ def _makeMVAPlots(num, hp=False):
                                   _makeDistSimPlots("3Dlayer"   , "3D layers"   , common=dict(xmin=_min3DLayers, xmax=_max3DLayers)),
                                   ncols=2, legendDy=_legendDy_4rows,
 )
-_extDistSimPosDeltaR = PlotGroup("distsimPosDeltaR",
+_extDistSimPos = PlotGroup("distsimPos",
                                  _makeDistSimPlots("vertpos", "vert r (cm)", common=dict(xlog=True)) +
                                  _makeDistSimPlots("zpos"   , "vert z (cm)") +
-                                 _makeDistSimPlots("simpvz" , "Sim. PV z (cm)", common=dict(xmin=_minZ, xmax=_maxZ)) +
-                                 _makeDistSimPlots("dr"     , "min #DeltaR", common=dict(xlog=True)),
-                                 ncols=2, legendDy=_legendDy_4rows,
+                                 _makeDistSimPlots("simpvz" , "Sim. PV z (cm)", common=dict(xmin=_minZ, xmax=_maxZ)),
+                                 ncols=3,
+)
+_extDistSimDeltaR = PlotGroup("distsimDeltaR",
+                                 _makeDistSimPlots("dr"     , "min #DeltaR", common=dict(xlog=True)) +
+                                 _makeDistSimPlots("drj"     , "#DeltaR(track, jet)", common=dict(xlog=True)),
```

Comment:
```suggestion
                                 _makeDistSimPlots("drj"     , "#DeltaR(TP, jet)", common=dict(xlog=True)),
```
would be more correct